src/network-validation/network-validation.constants.ts

```diff
@@ -0,0 +1,4 @@
+export const MODULE_ADDRESSES_FOR_CHAINS = {
```

Comment:
Not clear which module is it . also we use contract addresses for lido-nestjs-modules

Reply:
Agree. Will rename it to `CURATED_MODULE_ADDRESSES_FOR_CHAINS`.